lib/uuid/uuid.go

```diff
@@ -0,0 +1,11 @@
+package uuid
```

Comment:
This file and package doesn't look very useful.

Reply:
this is just to streamline the use of uuid that we use the same google's package across the code.